Tests/Fluent.Tests/Compute/VirtualMachineManagedServiceIdentityOperationsTests.cs

```diff
@@ -148,14 +148,16 @@ public void CanSetMSIOnNewVMWithRoleAssignedToCurrentResourceGroup()
 
 
                     var authenticatedClient = TestHelper.CreateAuthenticatedClient();
+                    // NOTE: Comment out since the below code need external tennat.
```

Comment:
Can you add a Todo instead of note, and add a GitHub issue to enable the following code once the subscription we use to record these tests supports the functionality?